packages/plugin-html-keyboard-response/src/index.ts

```diff
@@ -152,6 +152,60 @@ class HtmlKeyboardResponsePlugin implements JsPsychPlugin<Info> {
       this.jsPsych.pluginAPI.setTimeout(end_trial, trial.trial_duration);
     }
   }
+
+  simulate(
+    trial: TrialType<Info>,
+    simulation_mode,
+    simulation_options: any,
+    load_callback: () => void
+  ) {
+    if (simulation_mode == "data-only") {
+      load_callback();
+      this.simulation_data_mode(trial, simulation_options);
+    }
+    if (simulation_mode == "visual") {
+      this.simulation_visual_mode(trial, simulation_options, load_callback);
+    }
+  }
+
+  private simulation_create_data(trial: TrialType<Info>, simulation_options) {
```

Comment:
`create_simulation_data`? Oh, I see, `simulation_` was meant as a general prefix for all simulation-related methods. Hm, I still prefer imperative function names :smile: